spec/cli/yardoc_spec.rb

```diff
@@ -510,10 +510,23 @@ def foo; end
       optsdata = String.new("foo bar")
       expect(optsdata).to receive(:shell_split)
       expect(File).to receive(:read_binary).with("test").and_return(optsdata)
+      allow(optsdata).to receive(:gsub).and_return(optsdata)
       @yardoc.options_file = "test"
       @yardoc.run
     end
 
+    it "ignores .yardopts tokens that are comments" do
+      optsdata = String.new("--one-file --locale es # --locale en\n// --title not_my_title")
```

Comment:
This test should not pass, see above. Having "#" as an argument to YARD is completely valid.